### ISSUE TITLE
mpd fails to install boost-fiber and boost-locale dependencies

### ISSUE THREAD

**Issue by Be-ing** [openwrt/packages#3449]
Maintainer: @thess , @ClaymorePT 
Environment: Linksys WRT1200AC, LEDE build

Description:
```
# opkg install mpd
Installing mpd-full (0.19.19-1) to root...
Downloading http://downloads.lede-project.org/snapshots/packages/arm_cortex-a9_vfpv3/packages/mpd-full_0.19.19-1_arm_cortex-a9_vfpv3.ipk.
Collected errors:
 * satisfy_dependencies_for: Cannot satisfy the following dependencies for mpd-full:
 *      boost-fiber *   boost-locale * 
 * opkg_install_cmd: Cannot install package mpd.
```

Related: issue #3239 and PR #3266

**Comment by ClaymorePT** [comment]
Boost.Fiber is marked as BROKEN due to issues in the Boost.Build. 
This has been reported upstream. https://github.com/boostorg/build/issues/147

**Comment by thess** [comment]
MPD didn't require these before boost 1.62 update. I'll see what I can do short of rolling back boost to 1.61.

**Comment by ClaymorePT** [comment]
Could it be this piece of code in the makefile?
`		$(foreach lib,$(BOOST_LIBS), \
			config PACKAGE_boost-$(lib)
			prompt "Boost $(lib) library."
			$(if $(findstring locale,$(lib)),depends on @BUILD_NLS,)
			$(if $(findstring fiber,$(lib)),depends on BROKEN,)
		)
`

**Comment by thess** [comment]
@ClaymorePT - check out attempt to fix this #3458

**Comment by thess** [comment]
MPD and BOOST now building and producing good packages - closing this

See 8a98b22a00b046c85a09076bff795b7cb3bc0e8f
